lib/src/Pos/Diffusion/Full/Types.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE CPP           #-}
+{-# LANGUAGE TypeOperators #-}
+
+-- Copy of the "work mode" motif but for the diffusion layer. Collects all the
+-- constraints that we expect on a monad capable of carrying the diffusion
+-- layer.
+
+module Pos.Diffusion.Full.Types
```

Comment:
This module seems out of place in this PR -- it's not mentioned anywhere else in the changes